packages/story-editor/src/components/mediaRecording/audio.js

```diff
@@ -0,0 +1,119 @@
+/*
+ * Copyright 2022 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * External dependencies
+ */
+import { useEffect, useRef, useState } from '@googleforcreators/react';
+import styled from 'styled-components';
+import PropTypes from 'prop-types';
+
+const AudioWrapper = styled.div`
+  background: #26292a;
+  border-radius: 5px;
+  width: 100%;
+  height: 100%;
+`;
+
+export const Audio = ({ liveStream }) => (
+  <AudioWrapper>
+    <AudioAnalyser source={liveStream} />
+  </AudioWrapper>
+);
+
+Audio.propTypes = {
+  liveStream: PropTypes.object,
+};
+
+const AudioAnalyser = (props) => {
```

Comment:
```suggestion
const AudioAnalyser = ({ source }) => {
```
Nit pick, but props isn't very clear here.

Reply:
Yeah, I came up with a better naming 👍🏻